pkg/ui/src/views/login/loginContainer.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { RouterState } from "react-router";
+
+import LoginPage from "src/views/login/loginPage";
+import Layout from "src/views/app/containers/layout";
+
+interface LoginContainerState {
+  loggedIn: boolean;
+}
+
+class LoginContainer extends React.Component<RouterState, LoginContainerState> {
+  constructor(props: RouterState) {
+    super(props);
+    // TODO(vilterp): the user might already have a valid session. how do we check that?
+    // maybe do some templating on the backend and add it in.
```

Comment:
Local storage might be an answer, with a fallback on authorization failure.  A quick outline:

1. This needs to be in the redux state anyway - move this there.
2. After creating the store for the actual page, initialize it with one-time code that checks local storaage and, if it finds the logged-in flat, updates the redux state before the initial rendering.
3. When the login method actually succeeds, write to this local state after updating the redux state.
4. If a user method gets an authorization error, we need to clear this logged-in state if it exists.

You might see if there is some blessed method of synchronizing pieces of the redux state with local storage, but I doubt it's especially tricky.  #4 seems like the hard part.

Reply:
Didn't think of local storage, but it makes sense. My plan had been to make `index.html` a Go template into which I would dump `window.loggedInUser = {{ loggedInUser }}` or similar. However, `index.html` is not a nice little static file I can make into a template; it's generated through webpack (ultimately from this: https://github.com/jaketrent/html-webpack-template/blob/master/index.ejs)

So, local storage could work nicely. Will probably move forward with that unless @couchand has objections.